packages/backend/src/server/api/ApiCallService.ts

```diff
@@ -108,9 +108,9 @@ export class ApiCallService implements OnApplicationShutdown {
 		const [path] = await createTemp();
 		await pump(multipartData.file, fs.createWriteStream(path));
 
-		const fields = {} as Record<string, string | undefined>;
+		const fields = {} as Record<string, unknown | undefined>;
```

Comment:
`unknown` は最基底型なので `unknown | undefined` はすなわち `unknown` を意味します。
`null` が入るかどうかで次のどちらかになります

```suggestion
		const fields = {} as Record<string, unknown>;
```

```suggestion
		const fields = {} as Record<string, {} | undefined>;
```

Reply:
このあと値として使用する `MultipartValue.value` の定義を見るとデフォルトの型に `unknown` が指定されているため、 `unknown` の方を採用しようと思います。
https://github.com/fastify/fastify-multipart/blob/ce61100f93fb5a5cfe8ece49505617581cee0423/index.d.ts#L94